_educators/kevin_nelson.md

```diff
@@ -0,0 +1,22 @@
+---
+# Required:
+title: "Kevin Nelson"     # first (middle) last
+country: US              # your country of residence (2 letters)
+roles: [Instructor]              # subset of [facilitator, instructor, mentor], can stay empty ([])
```

Comment:
Currently this is still case sensitive, but I can also fix that later for you after merging this PR :)

Reply:
So shall I just merge this now, right?

Reply:
Yes, actually it already works as is, will just fix it later for consistency